src/nodes/network-requester-setup.md

```diff
@@ -1,3 +1,28 @@
+
```

Comment:
This should be in its own subsection far further down the page - something like 'updating to v1.1.10'. 

There is also something wrong with the formatting here which needs sorting out. 

![image](https://user-images.githubusercontent.com/33262279/220861098-a51b4040-8c46-4732-8cd6-df4bc48c7068.png)